### PR TITLE
Media gallery UI update

### PR DESCRIPTION
**Pull request by bmarty**
## Content

Small change on the media gallery following internal sync.
Figma: https://www.figma.com/design/Ni6Ii8YKtmXCKYNE90cC67/Timeline-(new)?node-id=1732-80010

Also enable the feature in production.

## Motivation and context

Improve UI rendering

## Screenshots / GIFs

See recorded one.

## Tests

- Open the gallery and some media

## Tested devices

- [ ] Physical
- [x] Emulator
- OS version(s):

## Checklist

- [ ] Changes have been tested on an Android device or Android emulator with API 23
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
- [ ] Pull request is based on the develop branch
- [ ] Pull request title will be used in the release note, it clearly define what will change for the user
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [ ] You've made a self review of your PR

### CODE REVIEW

File: libraries/mediaviewer/impl/src/main/kotlin/io/element/android/libraries/mediaviewer/impl/gallery/ui/VideoItemView.kt [hunk 1]
```diff
@@ -84,6 +85,14 @@ private fun VideoInfoRow(
...
         modifier = modifier
             .fillMaxWidth()
+            .background(
+                brush = Brush.verticalGradient(
+                    colors = listOf(
+                        Color.White.copy(alpha = 0f),
+                        Color.White,
+                    )
+                )
+            )
```
Comment (bmarty) [inline libraries/mediaviewer/impl/src/main/kotlin/io/element/android/libraries/mediaviewer/impl/gallery/ui/VideoItemView.kt:95]: I will update this, it has to render correctly on light theme too.

**Review by ganfra — APPROVED** [review]
Nice, thanks!

File: libraries/mediaviewer/impl/src/main/kotlin/io/element/android/libraries/mediaviewer/impl/gallery/ui/VideoItemView.kt (hunk 1 above)
Comment (ganfra) [reply libraries/mediaviewer/impl/src/main/kotlin/io/element/android/libraries/mediaviewer/impl/gallery/ui/VideoItemView.kt:95]: Why do we need this at all?

Comment (bmarty) [reply libraries/mediaviewer/impl/src/main/kotlin/io/element/android/libraries/mediaviewer/impl/gallery/ui/VideoItemView.kt:95]: It's a bg for the info (picto + duration) displayed above the video thumbnail.

**Pull request merged**